### PR TITLE
Explain LE backoff

### PR DESCRIPTION
**Pull request by johnsudaar**

### CODE REVIEW

File: _posts/platform/app/2000-01-01-ssl.md [hunk 1]
```diff
@@ -43,6 +43,12 @@ If you forget to renew your custom certificate and it expires, or if you delete
 Encrypt certificate will automatically and immediately replace it. Hence your application will
 always be available using HTTPS.
 
+If you're DNS configuration is not valid or if we fail to generate a
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-ssl.md]:
```suggestion
If youre DNS configuration is not valid or if we fail to generate a
```

File: _posts/platform/app/2000-01-01-ssl.md [hunk 2]
```diff
@@ -43,6 +43,12 @@ If you forget to renew your custom certificate and it expires, or if you delete
...
+certificate the platform will automatically retry it later. The time between those tries
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-ssl.md]:
```suggestion
certificate, the platform will automatically retry to generate it later. The time between those tries
```

File: _posts/platform/app/2000-01-01-ssl.md [hunk 3]
```diff
@@ -43,6 +43,12 @@ If you forget to renew your custom certificate and it expires, or if you delete
...
+is determined using an exponential backoff algorithm (we will wait 1mn for the
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-ssl.md]: Long parenthesis are not advised. In this case, it is just another sentence, no need for parenthesis

File: _posts/platform/app/2000-01-01-ssl.md [hunk 4]
```diff
@@ -43,6 +43,12 @@ If you forget to renew your custom certificate and it expires, or if you delete
...
+first retry, 2 for the second, 4 for the third and so on). This algorithm is
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-ssl.md]:
```suggestion
first retry, 2 minutes for the second, 4 minutes for the third and so on). This algorithm is
```

File: _posts/platform/app/2000-01-01-ssl.md [hunk 5]
```diff
@@ -43,6 +43,12 @@ If you forget to renew your custom certificate and it expires, or if you delete
...
+capped at a minimum of 1 try per day.
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-ssl.md]: Are you sure `capped` exists?

Comment (EtienneM) [inline _posts/platform/app/2000-01-01-ssl.md]: ~1 try~ one try

Comment (johnsudaar) [reply _posts/platform/app/2000-01-01-ssl.md]: Yep

File: _posts/platform/app/2000-01-01-ssl.md [hunk 6]
```diff
@@ -43,6 +43,12 @@ If you forget to renew your custom certificate and it expires, or if you delete
 Encrypt certificate will automatically and immediately replace it. Hence your application will
 always be available using HTTPS.
 
+If your DNS configuration is not valid or if we fail to generate a
+certificate the platform will automatically retry to generate it later.
```
Comment (EtienneM) [inline _posts/platform/app/2000-01-01-ssl.md]:
```suggestion
certificate, the platform will automatically retry to generate it later.
```

You didn't take all my suggestion ^^

**Pull request merged**